client/core/bot.go

```diff
@@ -742,7 +742,7 @@ func oracleMarketReport(ctx context.Context, b, q *SupportedAsset, log dex.Logge
 	// running on the same market.
 	var rawMarkets []*coinpapMarket
 	url := fmt.Sprintf("https://api.coinpaprika.com/v1/coins/%s/markets", baseSlug)
-	if err := getInto(ctx, url, &rawMarkets); err != nil {
```

Comment:
We could even hit the rate limit faster if there is a running bot using any of the fiat rate sources. We can consider sources that place limits only on a minute-by-minute basis.